go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -159,10 +159,6 @@ type ActionAgent struct {
 	// instead of asking mysqld what port it's serving on.
 	mysqlAdvertisePort int32
 
-	// initReplication remembers whether an action has initialized
-	// replication.  It is protected by actionMutex.
-	initReplication bool
```

Comment:
Looks like this was being assigned but never read? Just making sure I understand why it's being removed.

Reply:
Yes.